entry_tasks/4_raise_numbers/exponentiation.py

```diff
@@ -0,0 +1,6 @@
+if __name__ == '__main__':
+    print("Hello!\nI can only raise to higher power.\n")
+    x = input('Enter base:')
+    y = input('Enter power:')
+    result = int(x) ** int(y)
+print("Result: " + str(result))
```

Comment:
Looks good, but `print("Result: " + str(result)) ` should be within main block as well
Also, +1 empty line in the end of file should be paced (standard)